client/src/pages/Translations.jsx

```diff
@@ -0,0 +1,39 @@
+// @flow
+
+import React from 'react';
+import { Tabs, Layout, Row } from 'antd';
+
+import '../css/Translations.css';
```

Comment:
Since we don't have a CSS file for Translations, we can keep this as `import '../css/AdminResourceManager.css';` - also because we are using a `className` from there (`"admin-resource-form"`)